src/setupWorker/stop/createStop.ts

```diff
@@ -9,5 +9,6 @@ export const createStop = (context: ComposeMocksInternalContext) => {
     // Use this an an explicit way to stop the mocking, while preserving
     // the worker-client relation. Does not affect the worker's lifecycle.
     context.worker?.postMessage('MOCK_DEACTIVATE')
+    context.removeAllListeners()
```

Comment:
Let's nest everything related to the events/listeners management in its own key (i.e. `context.events`)?
So then the methods we call don't read like it's a `context` method, but it's key method?

```
context.events.add(target, event, handler)
context.events.removeAll()
```